hapi-fees/src/test/java/com/hedera/services/usage/token/TokenCreateUsageTest.java

```diff
@@ -126,13 +127,15 @@ private long baseSize() {
 				+ FeeBuilder.getAccountKeyStorageSize(adminKey)
 				+ FeeBuilder.getAccountKeyStorageSize(wipeKey)
 				+ FeeBuilder.getAccountKeyStorageSize(freezeKey)
-				+ FeeBuilder.getAccountKeyStorageSize(supplyKey);
+				+ FeeBuilder.getAccountKeyStorageSize(supplyKey)
+				+ memo.length();
 	}
 
```

Comment:
A test for expected delta when a memo is set would be good right ?

Reply:
I believe this test will fail if the memo bytes aren't taken into account, no?

Reply:
Ahh, my bad. I meant the other way around. We now loose the previous test where we are not setting a memo field and verifying the fee calculation metrics right. would mean the same when memo length is 0.